src/main/java/appeng/core/sync/packets/PacketCompassRequest.java

```diff
@@ -36,12 +36,12 @@
 public class PacketCompassRequest extends AppEngPacket implements ICompassCallback
 {
 
-	public final long attunement;
-	public final int cx;
-	public final int cz;
-	public final int cdy;
+	final long attunement;
```

Comment:
Everything related to networking/packets is due to be replaced with the network refactoring. I simply did not spend that much time in this package besides what tool support did automatically (lower the visibility of stuff when possible) as this is already done with the other PR.